app/data_model/list_store.py

```diff
@@ -12,24 +13,80 @@ def random_string(length):
     return ''.join(random.choice(ascii_letters) for _ in range(length))
 
 
+class ListRepresentation:
+    def __init__(self, list_items: List = None, context: Mapping = None):
+        if not list_items:
+            list_items = list()
+        if not context:
+            context = dict()
+
+        self.list_items = list_items
+        self.context = context
```

Comment:
Instead of the ifs?
```python
class ListRepresentation:
    def __init__(self, list_items: List = None, context: Mapping = None):
        self.list_items = list_items or []
        self.context = context or {}

```

Reply:
Fixed.